src/presentation/http/router/note.test.ts

```diff
@@ -385,6 +385,105 @@ describe('Note API', () => {
     test.todo('Returns 400 when parentId has incorrect characters and lenght');
   });
 
+  describe('DELETE /note/:notePublicId', () => {
+    test('Returns 200 status and "true" if note was removed successfully', async () => {
+      const accessToken = global.auth(1);
+      const correctID = 'Pq1T9vc23Q';
+
+      let response = await global.api?.fakeRequest({
+        method: 'DELETE',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/note/${correctID}`,
+      });
+
+      expect(response?.statusCode).toBe(200);
+
+      expect(response?.json()).toStrictEqual({ isDeleted: true });
+
+      response = await global.api?.fakeRequest({
+        method: 'DELETE',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/note/${correctID}`,
+      });
+
+      expect(response?.statusCode).toBe(406);
+
+      expect(response?.json()).toStrictEqual({ message: 'Note not found' });
```

Comment:
This should be in separate test case I suppose

Reply:
Do you mean that it's worth returning to the way it was before?
https://github.com/codex-team/notes.api/pull/177/commits/25adbc790db9db0bb80307913e653bb2186b016b